test/node/common-safe-require.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const tap = require('tap');
+
+const common = require('../../lib/common');
+
+const existingModule = 'tap';
```

Comment:
Maybe it would be better to require a core module here, but this works for me too.